### ISSUE TITLE
Blocks on the horison NOT FIXED by removing hidable flag

### ISSUE THREAD

**Issue by dadus33** [marcelo-mason/PreciousStones#853]
This issue has been flagged before but all the other tickets have been closed. In your last reply, you advised us to remove the hidable flag from them, and you closed the ticket without waiting for an answer. I tried that, and it didn't work. Here's my whole config: http://pastebin.com/eZg18XgX.
Here's how it looks like:
![issue](http://i.imgur.com/4pyJ8NW.png)
Removing the hidable flag or just resetting the database won't work, I tried both. Please look into it, my players keep asking when will be the bug fixed.
Thanks!

**Comment by marcelo-mason** [comment]
You'll have to remove the flag from the existing blocks as well.

**Comment by dadus33** [comment]
I did thtat, and here's what I got:
![image](http://prntscr.com/7lha5f)
I can still see the stones on the horizon.

**Comment by dadus33** [comment]
Please...

**Comment by dadus33** [comment]
Ignoring those bugs won't help anyone...

**Comment by Taybosquee** [comment]
@dadus33 mentioned teleportation in my similar but seemingly different issue (#859) so I figured I'd test it. Without fail, teleportation always triggers this bug when teleporting into a field that has been hidden. Fields with the hidable flag that have been hidden do cause this, but it's very possible that it's not the only thing that can considering this ticket.
I'm adding this here as it relates to the now closed #834 #837 which seem to be the same issue but didn't mention teleportation.

A temporary but not very practical solution I found for both my issue and this is re-connecting to the server. That seems to fix it, if only temporarily.
